concordia/urls.py

```diff
@@ -12,6 +12,7 @@
 from . import converters, views
 
 register_converter(converters.UnicodeSlugConverter, "uslug")
+register_converter(converters.ItemSlugConverter, "islug")
```

Comment:
I'd be included to give this a longer name since this is really saying that the value is something other than a slug. Maybe just go with `item_id`?

Reply:
Done